int32[] frame_id1
int32[] frame_id2
int32[] kp_id1
int32[] kp_id2

